mypyc/codegen/emitmodule.py

```diff
@@ -386,8 +386,11 @@ def load_scc_from_cache(
     Arguments and return are as compile_scc_to_ir.
     """
     cache_data = {
-        k.fullname: json.loads(
-            result.manager.metastore.read(get_state_ir_cache_name(result.graph[k.fullname]))
+        k.fullname: cast(
+            Any,
```

Comment:
I suppose you could also cast to `Mapping[Literal["ir"], str]`, or even better, to `optype.CanGetitem[Literal["ir"], str]`

Reply:
yeah, but it's not my code :D